e2e/playwright/point-click.spec.ts

```diff
@@ -97,6 +107,102 @@ test.describe('Point-and-click tests', () => {
     })
   })
 
+  test('Verify in-pipe extrudes in bracket can be edited', async ({
```

Comment:
New test to showcase the intent here: edit the extrudes in the real-life onboarding bracket